src/lib/constants/element/datepicker.py

```diff
@@ -0,0 +1 @@
+SELECTOR = "a.ui-state-default"
```

Comment:
My approach would be to have them all at one place, and when it becomes too much, we would partition these variables into different files. It's a bit more agile approach to the same problem and it was just a suggestion. Please ignore my comment if you think this will be better in the long run.